admin/influx/create_test_db.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", ".."))
+from influxdb import InfluxDBClient
+import config
+
+try:
+    i = InfluxDBClient(host=config.INFLUX_HOST, port=config.INFLUX_PORT, database=config.INFLUX_TEST_DB)
+    i.create_database(config.INFLUX_TEST_DB)
+except Exception as e:
+    print("Creating influx DB failed: ", e)
```

Comment:
This is what I was alluding to in the PR description -- this is discombobulated and with the volume-less containers, the -test portions of the setup script will simply go away once LB-116 is addressed. But, I would rather not clean any of that up right this second.